Formula/d/driftctl.rb

```diff
@@ -6,6 +6,11 @@ class Driftctl < Formula
   license "Apache-2.0"
   head "https://github.com/snyk/driftctl.git", branch: "main"
 
+  livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
```

The `stable` URL is a tag archive (rather than a release artifact), so we should explain the use of `GithubLatest` here.